pkg/proof/querier.go

```diff
@@ -156,3 +162,10 @@ func ParseNamespace(rawShares []shares.Share, startShare, endShare int) (appns.N
 	}
 	return startShareNs, nil
 }
+
+func safeConvert(x int64) (int, error) {
```

Comment:
[optional]
Guess it would make sense for this function to specify exactly which conversion it will do in the title so that if we need other conversions, we don't end up renaming this one.

```suggestion
func safeConvertInt64ToInt(x int64) (int, error) {
```

Reply:
Resolved via fc530d880eb6b484dfcdc126f3a7696d96750e1f